homeassistant/components/zha/strings.json

```diff
@@ -86,11 +86,11 @@
       },
       "intent_migrate": {
         "title": "Migrate to a new radio",
-        "description": "Your old radio will be factory reset.  If you are using a combined Z-Wave and Zigbee adapter like the HUSBZB-1, this will only reset the Zigbee portion.\n\nDo you wish to continue?"
+        "description": "Backup to file will performed and your old radio will then be reset to its default factory configuration.  If you are using a combined Z-Wave and Zigbee adapter like the HUSBZB-1, this will only reset the Zigbee portion.\n\nDo you wish to continue?"
       },
       "instruct_unplug": {
         "title": "Unplug your old radio",
-        "description": "Your old radio has been reset.  If the hardware is no longer needed, you can now unplug it."
+        "description": "Your old radio has been reset.  If the old hardware adapter is no longer needed, you can now unplug it. Backup was saved to file if you should wish to revert back to your old radio."
```

Comment:
Can I suggest you add an instruction to plug in the new hardware. Since the next screen in the config flow is the auto-detection of connected hardware if the user hasn't plugged in the new hardware before clicking next the hardware won't autodetect. (I had this issue)